src/main/java/org/broadinstitute/hellbender/dev/pipelines/bqsr/ReadsFilter.java

```diff
@@ -0,0 +1,35 @@
+package org.broadinstitute.hellbender.dev.pipelines.bqsr;
+
+import com.google.api.services.genomics.model.Read;
+import com.google.cloud.dataflow.sdk.transforms.DoFn;
+import com.google.cloud.genomics.gatk.common.GenomicsConverter;
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMRecord;
+import org.broadinstitute.hellbender.engine.filters.ReadFilter;
+
+/**
+ * A simple filter for reads. Pulled into its own class due to Java limitations.
+ */
+public final class ReadsFilter extends DoFn<Read, Read>   {
```

Comment:
Also, when this eventually moves out of dev, we will need to do a refactoring pass to bring sanity to all of these class names. Now we have, eg., both `ReadsFilter` and `ReadFilter`.